### ISSUE TITLE
Skip to main content needed

### ISSUE THREAD

**Issue by IdaAkiwumi** [chnm/rrchnm2016#82]
I just noticed after tabbing consistently, that there is no Skip to main content box that comes up for the user to be able to skip navigation. There's more info here: https://accessibility.digital.gov/ux/navigation-shortcuts/.

**Comment by kimisgold** [comment]
* Reposition the "skip to content" link to avoid obscuring other content. Ida suggested creating vertical space at the top when the link has focus.
* Make sure screen readers have content to read when the page jumps to the content area.

**Comment by IdaAkiwumi** [comment]
To add to the above, this is where the voice over utility focuses after skip to main content has been activated. It visually moves down to the main content (which is good), but doesn't grasp it for reading purposes. The black rectangle (this is how VoiceOver Utility reads the text) above "25 Years.." should be surrounding it instead.

![Screen Shot 2020-10-06 at 10 52 50 AM (2)](https://user-images.githubusercontent.com/618307/95222934-08a74b80-07c7-11eb-9492-538516c7d923.png)

**Comment by kimisgold** [comment]
Commit 92c5648 adds javascript that should bring the focus from the skipnav link to the first heading of the content section.

**Comment by kimisgold** [comment]
This is live and ready for another look.

**Comment by IdaAkiwumi** [comment]
Looks good!
